test/fw/bin/pbs_benchpress

```diff
@@ -449,6 +458,35 @@ if __name__ == '__main__':
         db.set_data(dbtype, dbname, dbaccess)
         data.set_data(post_data_dir, max_postdata_threshold)
         plugins = (loader, testtags, runner, db, data)
+    if extra_plugins:
+        for plugin in extra_plugins.split(','):
+            if '=' not in plugin:
+                _msg = 'Invalid value (%s)' % (plugin)
+                _msg += ' provided in extra-plugins, it should be key value'
+                _msg += ' pair where key is module name and value is class'
+                _msg += ' name of plugin'
+                logging.error(_msg)
+                sys.exit(1)
+            mod, clsname = plugin.split('=')
```

Comment:
Consider doing split('=', 1) since you are only accepting two arguments.  If someone did foo=bar=baz, this will barf.

Reply:
I like using split('=', 1) but here we are safe by using just split('=') because '=' is not allowed in module name or class name in python.
```
xxx@xxx:~/code/myplugins$ vi 't=.py'
xxx@xxx:~/code/myplugins$ python
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import t=
  File "<stdin>", line 1
    import t=
            ^
SyntaxError: invalid syntax
>>> import 't='
  File "<stdin>", line 1
    import 't='
              ^
SyntaxError: invalid syntax
>>> exit()
xxx@xxx:~/code/myplugins$ vi myplugin.py
xxx@xxx:~/code/myplugins$ python
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import myplugin
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "myplugin.py", line 9
    class My=Info(Plugin):
            ^
SyntaxError: invalid syntax
>>>
```
But if still think we should use split('=', 1) then I am happy to make change.